### PR TITLE
Fix error when creating NetCDF resources using REST API

### PR DESCRIPTION
**Pull request by selimnairb**
Addresses #569.

### CODE REVIEW

**Comment by selimnairb** [comment]
All unit tests that passed under develop pass under this branch.

File: hs_core/views/resource_rest_api.py [hunk 1]
```diff
@@ -283,7 +283,7 @@ def create(self, request, *args, **kwargs):
                 raise ValidationError(detail={'file': 'Multiple file upload is not allowed on resource creation. Add additional files after the resource is created.'})
             # Place files into format expected by hydroshare.utils.resource_pre_create_actions and
             # hydroshare.create_resource, i.e. a tuple of django.core.files.uploadedfile.TemporaryUploadedFile objects.
-            files = (request.FILES['file'],)
+            files = [request.FILES['file'],]
         else:
             files = ()
```
Comment (hyi) [inline hs_core/views/resource_rest_api.py]: @selimnairb Should empty initialization to files in else phrase be changed to [] if files are changed from tuple to list?

Comment (ironfroggy) [inline hs_core/views/resource_rest_api.py]: i would say yes, if for no reason but consistency.

**Comment by selimnairb** [comment]
@hyi @ironfroggy Good point.  I know this may be un-Pythonic, but long term, I would prefer to refactor the resource creation code so that files is required to be a mutable sequence type (i.e. list).

**Comment by selimnairb** [comment]
Anyone object to merging this?

**Comment by hyi** [comment]
+1

**Comment by pkdash** [comment]
+1

**Pull request merged**